src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+
+const sortSex = (gender) => {
+  if (gender === 'm') {
+    return 'wife';
+  }
+
+  return 'husband';
+};
+
+export const Person = ({ person }) => (
+  <div className="Person">
```

Comment:
it's a good practice to use object destructing
```suggestion
export const Person = ({ person }) => {
  const { age, ...} = person
  return (
    <div className="Person">
```